src/anchor-navigation/interfaces.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { BaseComponentProps } from '../internal/base-component';
+import { CancelableEventHandler, NonCancelableEventHandler } from '../internal/events';
+
+export interface AnchorNavigationProps extends BaseComponentProps {
+  /**
+   * Adds `aria-labelledby` to the component.
+   *
+   * Use this property for identifying the header or title that labels the anchor navigation.
+   *
+   * To use it correctly, define an ID for the element you want to use as label and set the property to that ID.
+   */
+  ariaLabelledby?: string;
+
+  /**
+   * List of anchors.
+   */
+  anchors: AnchorNavigationProps.Anchor[];
+
+  /**
+   * Specifies the active anchor href.
+   * For using the component in a controlled manner, use together with 'disableTracking'.
+   */
+  activehref?: string;
```

Comment:
Is the "h" intentionally lowercase?

The Side navigation component has `activeHref` with uppercase "H".

Reply:
Not intentional! Fixed it.